cmd_nodes:
  - topic_name:  "can0"
    timeout:     1
    priority:    1
    short_desc:  "controllers cmd_vel "

  - topic_name:  "cmd_vel"
    timeout:     1
    priority:    10
    short_desc:  "Navigation stack controller"

  - topic_name:  "/dev/input/js0"
    timeout:     1
    priority:    2
    short_desc:  "F710 joy"

  - topic_name:  "keyboard_cmd_vel"
    timeout:     1
    priority:    11
    short_desc:  "keyboard"

  # - topic_name:  "cmd_vel_filtered"
  #   timeout:     1
  #   priority:    3
  #   short_desc:  "navigation filtered"

  - topic_name:  "web"
    timeout:     1
    priority:    20
    short_desc:  "web"

  - topic_name:  "heart_cmd_vel"
    timeout:     1
    priority:    100
    short_desc:  "heart"

# publisher:       "cmd_vel_rectified"
publisher:       "cmd_vel_mux_rectified"


joy_types:
  - joy_frame: "/dev/input/js0"
    max_linear_speed: 0.5
    max_angular_speed: 0.25
  - joy_frame: "/can0"
    max_linear_speed: 0.8
    max_angular_speed: 0.25
  - joy_frame: "/web"
    max_linear_speed: 0.5
    max_angular_speed: 0.25

